### PR TITLE
[GP]: Content for registration pages

### PR DESCRIPTION
**Pull request by tidybit**
New content for registration pages

### CODE REVIEW

File: app/uk/gov/hmrc/eeitt/views/verification_non_agent.scala.html [hunk 1]
```diff
@@ -73,45 +73,21 @@
     <h2>Problems signing in</h2>
 
     <details class="details">
-      <summary id="registration-link-container" role="button" aria-expanded="false">
-        Don't have an agent reference number
-      </summary>
+      <summary id="registration-link-container" role="button" aria-expanded="false">Don't have an Air Passenger Duty registration number</summary>
```
Comment (konradwudkowski) [inline app/uk/gov/hmrc/eeitt/views/verification_non_agent.scala.html]: Can we please keep it generic and not mention Air Passenger Duty?

File: app/uk/gov/hmrc/eeitt/views/formerror.scala.html [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <div id="errorMessage" class="flash error-summary error-summary--show">
-    <p>Something went wrong. Please try again.</p>
+    <p>Invalid registration number or business postcode. Try again.</p>
```
Comment (konradwudkowski) [inline app/uk/gov/hmrc/eeitt/views/formerror.scala.html]: 2 questions here: 
- Is this formerror.scala.html even used anywhere?
- Was removing 'please' intentional? (I'm not a content designer so just playing a devil's advocate)

File: app/uk/gov/hmrc/eeitt/views/verification_non_agent.scala.html [hunk 3]
```diff
@@ -73,45 +73,21 @@
...
-    </details>
-
-    <details class="details">
-      <summary role="button" aria-expanded="false">
-        Client doesn't have an Air Passenger Duty registration number
-      </summary>
-      <div class="details__inner" id="details-content-2">
-        <p class="panel-indent">
-          placeholder
+          Don't have a registration number. <a href="https://www.gov.uk/topic/business-tax">Choose your relevant tax and then follow the registration instructions.</a>
```
Comment (konradwudkowski) [inline app/uk/gov/hmrc/eeitt/views/verification_non_agent.scala.html]: Was it the intention to say _Don't have a registration number twice_ as in the attached screenshot?

image

File: app/uk/gov/hmrc/eeitt/views/formerror.scala.html (hunk 2 above)
Comment (konradwudkowski) [inline app/uk/gov/hmrc/eeitt/views/formerror.scala.html]: Did you check if formerror is used somewhere? If not can it be deleted?

File: app/uk/gov/hmrc/eeitt/views/verification_non_agent.scala.html [hunk 4]
```diff
@@ -32,86 +32,60 @@
...
-        @errorInline("registrationNumber", "Please enter a valid registration number", Seq("optional-extra-css-classes"))
+        @errorInline("registrationNumber", "Please enter your registration number", Seq("optional-extra-css-classes"))
         <input
         type="text"
         name="registrationNumber"
         id="registrationNumber"
         value="@enrollmentForm("registrationNumber").value"
         class="form-control form-control--block"
         required
-        pattern="^[a-zA-Z0-9]{15}$"
```
Comment (konradwudkowski) [inline app/uk/gov/hmrc/eeitt/views/verification_non_agent.scala.html]: You have removed validation here. Why? Can you also add server side validation for this field as it needs to be nonempty and we extract tax regime from it so if it has less than 4 chars extracting 3rd and 4th characters will fail...

Comment (tidybit) [inline app/uk/gov/hmrc/eeitt/views/verification_non_agent.scala.html]: Was removed as the play pattern we are using for client side validation does not appear to allow multiple user messages for different validation rules. We have a new generic user message now so will add client side string length validation back in.

**Pull request merged**